src/app/editor/components/conflicted-field/conflicted-field.component.spec.ts

```diff
@@ -54,6 +54,44 @@ describe('ConflictedFieldComponent', () => {
         expect(component).toBeTruthy();
     });
 
+
+    it('should render simple text', () => {
+        component.conflictedField = {
+            field: { type: 'string', name: 'somename' },
+            localValue: 'local value',
+            remoteValue: 'remote value',
+            overwrite: false,
+            conflictedFields: []
+        };
+        fixture.detectChanges();
+
+        const mineElement: DebugElement = fixture.debugElement.query(By.css('.mine-val'));
+        const theirElement: DebugElement = fixture.debugElement.query(By.css('.their-val'));
+
+        expect(mineElement.nativeElement.innerHTML).toContain(component.conflictedField.localValue);
+        expect(theirElement.nativeElement.innerHTML).toContain(component.conflictedField.remoteValue);
+    });
+
+
+    it('should render checkboxes for boolean conflicts', () => {
+        component.conflictedField = {
+            field: { type: 'boolean', name: 'somename' },
+            localValue: true,
+            remoteValue: false,
+            overwrite: false,
+            conflictedFields: []
+        };
+        fixture.detectChanges();
+
+        const mineElement: DebugElement = fixture.debugElement.query(By.css('.mine-val gtx-checkbox'));
+        expect(mineElement).toBeDefined();
+        expect(mineElement.attributes['ng-reflect-checked'].toString()).toEqual('true');
```

Comment:
Inspect the `Checkbox` instance `checked` property instead of the "ng-reflect..." by adding `.componentInstance` to the debugElement